tests/Spec/Data/VectorNonEmptySpec.hs

```diff
@@ -0,0 +1,33 @@
+{-|
+Module:      Spec.Data.Vector.NonEmptySpec
+Copyright:   (C) 2014-2017 Ryan Scott
+License:     BSD-style (see the file LICENSE)
+Maintainer:  Ryan Scott
+Stability:   Provisional
+Portability: GHC
+
+@hspec@ tests for 'Vector' types.
+-}
+module Spec.Data.VectorNonEmptySpec (main, spec) where
```

Comment:
For consistency with how `nonempty-vector` names its modules, let's name this module `Spec.Data.Vector.NonEmptySpec` instead.